packages/itwin/property-grid/README.md

```diff
@@ -201,32 +221,39 @@ Adding a custom settings menu item can be done as follows:
 
 Define a menu item component:
 
+<!-- [[include: [PropertyGrid.ExampleSettingsMenuItemImports, PropertyGrid.ExampleSettingsMenuItem], tsx]] -->
+<!-- BEGIN EXTRACTION -->
+
 ```tsx
+import { PropertyGridSettingsMenuItem } from "@itwin/property-grid-react";
+
 function ExampleSettingsMenuItem() {
   return (
     // render using `PropertyGridSettingsMenuItem` to get consistent style
-    <PropertyGridSettingsMenuItem
-      id="example"
-      onClick={() => {
-        console.log("Settings menu clicked!");
-      }}
-    >
+    <PropertyGridSettingsMenuItem id="example" onClick={() => {}}>
       Click me!
     </PropertyGridSettingsMenuItem>
   );
 }
 ```
 
+<!-- END EXTRACTION -->
+
 Provide it to the widget:
 
-```ts
-new PropertyGridUiItemsProvider({
-  propertyGridProps: {
-    settingsMenuItems: [(props) => <ExampleSettingsMenuItem />],
-  },
-});
+<!-- [[include: [PropertyGrid.PropertyGridWithSettingsMenuItemImports, PropertyGrid.PropertyGridWithSettingsMenuItem], tsx]] -->
+<!-- BEGIN EXTRACTION -->
+
+```tsx
+import { PropertyGridComponent } from "@itwin/property-grid-react";
+
+const MyPropertyGrid = () => {
```

Comment:
Can you update readme and test to define components using `function` instead of `const` to make it consistent? At the moment some of them are defined like `function Component() {` and some like `const Component = () => {`